frontend/src/pages/UserPage.jsx

```diff
@@ -97,6 +105,8 @@ export default function UserPage() {
     (friendIdList.includes(selectedUser?.id) ||
       selectedUser?.id === currentUser?.id);
 
+  const isMyPage = selectedUser && selectedUser?.id === currentUser?.id;
```

Comment:
optional chaining이라 
```suggestion
  const isMyPage = selectedUser?.id === currentUser?.id;
```
도 충분할 것 같아요!